js/tools/kermit.js

```diff
@@ -76,6 +79,26 @@ function grep (path, pattern) {
     });
 }
 
+// This hash is derived from sha1('link-table'), and for now is the hardcoded
+// location of link tables for all shards.
+const LINK_TABLE_ID = '4dba9091495e8f277893e0d400e9e092f9f6f551';
+function lookup (path, url) {
+    let shard = get_shard_for_path(path);
+    let table_record = shard.find_record_by_hex_name(LINK_TABLE_ID);
+    let dictionary;
+    if (table_record) {
+        dictionary = table_record.data.load_as_dictionary();
+    } else {
+        fail_with_message('No dictionary record found in this shard!');
+    }
+    let ekn_id = dictionary.lookup_key(url);
+    if (!ekn_id) {
```

Comment:
In if/else statements, I prefer not inverting the conditional. Makes it easier to read, IMO